saleor/product/migrations/0037_auto_20171113_0505.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we could have one migration file instead of:
`saleor/product/migrations/0036_auto_20171110_0602.py` and `saleor/product/migrations/0036_auto_20171110_0602.py`

Reply:
Did you mean `0037_xxxx`? Sure, I can merge migrations per appinto one. I wasn't paying much attention to it, as @koradon said we can always merge all migration files at the end of the PR.

Reply:
Sorry, I meant `0037`. Of course, you can do it later if you're still working on the PR. In general, the fewer migration files the better, so when we have similar migrations we should try to merge them.

Reply:
Migration files has been merged into on 0036.